samples/androidNativeActivity/settings.gradle

```diff
@@ -1,2 +1,20 @@
-includeBuild '../../shared'
-includeBuild '../../tools/kotlin-native-gradle-plugin'
+// Reuse Kotlin version from the root project.
```

Comment:
What about path to Kotlin/Native distribution?

Reply:
Good point. I will add handling for path.

Reply:
Fixed.